shopelectro/selenium/elements/product_card.py

```diff
@@ -0,0 +1,19 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class ProductCard:
+    """"Represent a product card of category page."""
+
+    def __init__(self, driver, card_number):
+        self.driver = driver
+        self.button_xpath = f'//*[@id="products-wrapper"]/div[{card_number}]/div[2]/div[5]/button'
+
+    def _get_element(self, xpath):
+        self.driver.wait.until(EC.visibility_of_element_located(By.XPATH, xpath))
```

Comment:
we are not needed to pass xpath as the arg, because we already have xpath as field of `self`

Reply:
we should pass `By.XPATH, xpath` pair as tuple as i know. Maybe it's not necessary, but we usually do like this